internal/cmd/about.go

```diff
@@ -40,14 +40,12 @@ func newTriangle(a, b, c *point3D, ch rune) *triangle {
 }
 
 func (t *triangle) Draw(surf *sprite.Surface) {
-	err := surf.Triangle(
+	// nolint
+	surf.Triangle(
```

Comment:
I think this should work to indicate "ignore error":
```suggestion
	_ = surf.Triangle(
```

Reply:
This worked 👍